Makefile

```diff
@@ -60,19 +60,15 @@ nix-purge: SHELL := /bin/sh
 nix-purge: ##@nix Completely remove the complete Nix setup
 	sudo rm -rf /nix ~/.nix-profile ~/.nix-defexpr ~/.nix-channels ~/.cache/nix ~/.status .nix-gcroots
 
-nix-add-gcroots: SHELL := /bin/sh
 nix-add-gcroots: ##@nix Add Nix GC roots to avoid status-react expressions being garbage collected
 	scripts/add-nix-gcroots.sh
 
-nix-update-npm: SHELL := /bin/sh
```

Comment:
Should we add a specific Nix shell target for these Make targets so we don't force a ton of stuff to be downloaded/built?

Reply:
You're right.